htdocs/societe/list.php

```diff
@@ -551,46 +551,46 @@
 if ($search_id > 0) {
 	$sql .= natural_search("s.rowid", $search_id, 1);
 }
-if ($search_nom) {
+if (!empty($search_nom)) {
```

Comment:
Did you get warning due to this line ?
$search_nom seems to be always defined (set at top of file) so i don't see why you should need this here.